plugins/languages/rust/rustaceanvim/default.nix

```diff
@@ -74,6 +70,9 @@ helpers.neovim-plugin.mkNeovimPlugin config {
               Note that if you supplied an attrset and not a function you need to set this attr set in:
                 `plugins.rustaceanvim.settings.server.default_settings.rust-analyzer'.
             '';
+        plugins.rustaceanvim.config.init = ''
```

Comment:
Will this help resolve the checkhealth error we've had ? 
```
- ERROR vim.g.rustaceanvim is set, but it was sourced after rustaceanvim was initialized.
```

Reply:
Would need to look into what that error really means, but it sounds like rustaceanvim is unhappy that we're setting `globals.rustaceanvim` before calling their `setup` function?

Reply:
Yeah, didn't know if adding this was setting up the global before setup, now. Or if we needed to move it to `pre` instead of init.

Reply:
Sorry, looking at their [readme](https://github.com/mrcjkb/rustaceanvim?tab=readme-ov-file#zap-quick-setup) they don't use the setup call and just use a filetype detection to initialize. Not sure why we'd be getting that health error, can just look at it separately.